CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ add_executable(mpm_point_generator ${mpm_point_gen_src})
 SET(test_src
   ${PROJECT_SOURCE_DIR}/tests/test.cc
   ${PROJECT_SOURCE_DIR}/tests/point_test.cc
+  ${PROJECT_SOURCE_DIR}/tests/io_test.cc
+  ${PROJECT_SOURCE_DIR}/tests/gmsh_test.cc
```

Comment:
This is a regression test and not a unit test and should therefore be built separately.